juniper_official/Solutions/EX-Platform/check-ethernet-mac-count-netconf.rule

```diff
@@ -0,0 +1,175 @@
+/*
+* Monitors the health of MAC count in ethernet switching and notifies
+* when anomalies are found
+* Two input control detection
+*
+* 1) vlan-id, is a regular expression that matches the
+* VLAN IDs that you would like to monitor. By default
+* it's '.*', which matches all VLAN IDs.
+*
+* 2) static-threshold, is the threshold that causes the rule to report
+* an anomaly. By default MAC limit threshold per VLAN is 800 . This
+* rule will set a dashboard color to red when MAC count exceeds the
+* threshold value. If dynamic threshold of MAC count is equal to 1,
+* the color is set to yellow. Otherwise it is set to green.
+*/
+healthbot {
+    topic ethernet.options {
```

Comment:
can we change topic to ethernet.statistics or better one?

Reply:
updated